planner/planner_API.go

```diff
@@ -13,22 +24,18 @@ import (
 	"github.com/weihesdlegend/Vacation-planner/solution"
 	"github.com/weihesdlegend/Vacation-planner/user"
 	"github.com/weihesdlegend/Vacation-planner/utils"
-	"html/template"
-	"io/ioutil"
-	"net/http"
-	"net/url"
-	"os"
-	"regexp"
-	"strconv"
-	"strings"
-	"time"
 )
 
 const (
 	ServerTimeout      = time.Second * 15
 	jobQueueBufferSize = 1000
 )
 
+var placeTypeToIconCss = map[POI.PlaceCategory](string){
+	POI.PlaceCategoryEatery: "restaurant",
```

Comment:
We should define a new Enum for the Icon CSS class names.

Reply:
Good idea, thanks. Will do

Reply:
A new Enum is added in commit 6088153